style/client.css

```diff
@@ -3399,7 +3427,7 @@ a.ilink.yours {
 .dark .utilichart .namecol,
 .dark .utilichart .pokemonnamecol,
 .dark .utilichart .movenamecol {
-	color: #DDD;
+	color: #FFF;
```

Comment:
Why?

Reply:
(updated) this color shouldn't change , see below